doc/tutorials/content/pcl_vcpkg_windows.rst

```diff
@@ -0,0 +1,81 @@
+.. _use_pcl_windows:
+
+Using PCL on windows with VCPKG and CMake
+------------------------------------
+
+This tutorial explains how to acquire Point Cloud Library on
+Microsoft Windows platforms using `VCPKG <https://github.com/microsoft/vcpkg>`_.
+
+.. image:: images/windows_logo.png
+   :alt: Microsoft Windows logo
+   :align: right
+
+.. contents::
+
+Requirements
+==================
+
+Download VCPKG sources to eg. *c:\\vcpkg* preferably by cloning their repository.
+
+Navigate to *c:\\vcpkg* in **powershell** and run 
+
+  .\\bootstrap-vcpkg.bat
+  
+This will download vcpkg.exe.
+
+Install PCL for usage
+==================
+
+Running the following command in powershell in the VCPKG directory,
+will install PCL with default options as well as default triplet type (ie. x86).
+
+  ./vcpkg install pcl
+  
+
+.. note::
+
+  If there are new features or bugfixes that is not yet part of a release,
+  you can try use --head, which downloads the master of PCL.
+  
+  - 2021-06-25 PCL 11(.1) doesn't support VTK 9, so if you need visualization module, use --head
+
+You can see the available options `here <https://github.com/microsoft/vcpkg/blob/030cfaa24de9ea1bbf0a4d9c615ce7312ba77af1/ports/pcl/CONTROL>`_
+
+To enable specific features, you need to write:
+
+  ./vcpkg install pcl[qt,vtk]
+
+And all features:
+
+  ./vcpkg install pcl[*]
+
+If you want to install with a different triplet type, the easiest way is:
+
+  ./vcpkg install pcl --triplet triplet_type, ie ./vcpkg install pcl --triplet x64-windows
+
+This will acquire all the dependencies, build them and place the binaries
+in vcpkg/installed/triplet_type/bin for release and vcpkg/installed/triplet_type/debug/bin for debug.
+
+Find PCL using CMake
+==================
+
+To use PCL in CMake project, take a look at Kunals Tyagi's minimal example `here <https://github.com/kunaltyagi/pcl-cmake-minimum>`_
+
+Use `CMake <https://cmake.org/download>`_ to configure your project and remember to pass **vcpkg\\scripts\\buildsystems\\vcpkg.cmake** as toolchain file
+to enable CMake to find all the dependencies installed with VCPKG.
+
+Install PCL dependencies for contributions
+==================
+
+If you want to contribute to PCL, the easiest way to get dependencies
+using vcpkg is to run the install command from our `docker file <https://github.com/PointCloudLibrary/pcl/blob/master/.dev/docker/windows/Dockerfile>`_
+
+  ./vcpkg install dependencies_here --triplet triplet_type
```

Comment:
Maybe add, as of 1.11.1, the dependencies are <A, B, C>

Reply:
I have added a list of all the dependencies and what they required for.